packages/docs-page/util/index.ts

```diff
@@ -1,5 +1,5 @@
-const REGEX =
-  /^v([0-9]+)\.([0-9]+)\.(x|[0-9]+)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$/i
```

Comment:
Are we potentially losing anything by simplifying this regex? 💭

Reply:
@zchsh @BRKalow the `REGEX` before this PR was potentially an oversight. It'd match semvers w/ RC suffixes, like `v1.2.3-rc.1`.

The `-rc.1` suffix is what we'd lose by simplifying the regex

Reply:
Makes sense! Just a future thought, if/when we support pre-releases we might have to bring this back (no action needed now)